src/vector/v.scatterplot/v.scatterplot.html

```diff
@@ -24,6 +26,30 @@ <h2>DESCRIPTION</h2>
 given by the user. So, if file_name = outputfile.png, the plot will be 
 saved as a PNG file.
 
+<p>
+A linear or polynomial trend line with user-defined degrees can be 
+drawn on top of the scatter/density plot. 
+
+<p>
+A confidence ellipse of the covariance of the two variables can be 
+plotted on top of the scatterplot, following the method described <a 
+href="https://carstenschelp.github.io/2018/09/14/Plot_Confidence_Ellipse_001.html">here</a>, 
+and using the code described <a 
+href="https://matplotlib.org/stable/gallery/statistics/confidence_ellipse.html">here</a>. 
+The radius of the ellipse can be controlled by n_sd which is the number 
+of standard deviations (SD). The default is 2 SD, which results in an 
+ellipse that encloses around 95% of the points. Optionally, separate 
+confidence ellipses can be drawn for groups defined in the column 
+<i>groups</i>. Groups can be assigned a random color, or a color based 
+on the RGB colors in a user-defined column. Note, all records in the 
+group should have the same color.
+
+<p>
+The user has the option to limit/expand the X-axis (<i>xlim</i>) and 
+Y-axis (<i>ylim</i>). This can e.g., make it easier to compare 
```

Comment:
```suggestion
The user has the option to limit/expand the X-axis (<em>xlim</em>) and 
Y-axis (<em>ylim</em>). This can e.g., make it easier to compare 
```

Compare: https://trac.osgeo.org/grass/wiki/Submitting/Docs#Markupstyleguide